guides/common/modules/proc_upgrading-smartproxy-server.adoc

```diff
@@ -139,3 +139,28 @@ include::snip_steps-needs-reboot.adoc[]
 ifdef::orcharhino,satellite[]
 . Optional: If you use custom repositories, ensure that you enable these custom repositories after the upgrade completes.
 endif::[]
+
+.Upgrading {SmartProxyServersTitle} using remote execution
+
+. Create a backup or take a snapshot.
```

Comment:
This sounds like a prerequisite.

Reply:
Agreed, will move this into the prerequisites when rearranging.